### DISCUSSION TITLE
Επαναχρησιμοποίηση υλικού

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#73] [Q&A] [answered]
Καλησπέρα,

Θα ήθελα να ρωτήσω αν κατά τη γραπτή εξέταση του Ιουνίου, και κατ' επέκταση στο Μέρος Β της 5ης εργασίας που αποτελεί προσομοίωσή της, μπορούμε να χρησιμοποιήσουμε μέρη από τις απαντήσεις των προηγούμενων εργασιών _μας_ (πχ άρθρα, κριτήρια, μεθόδους, περσόνες). Υποθέτω ότι αφού οι εξετάσεις είναι με ανοικτές σημειώσεις, έχουμε το δικαίωμα να ανατρέξουμε σε υπάρχον υλικό και να το αναπαράγουμε χωρίς να υπάρχει πρόβλημα "αντιγραφής", δεδομένου ότι δεν υφίσταται συνεργασία με άλλο άτομο.

Επίσης θα ήθελα να ρωτήσω στο Ερωτ. 1.2 του Μέρους Β για τα άρθρα: αυτά πρέπει να αναφέρονται σε μελέτες των κριτηρίων σχεδιασμού εφαρμογών όσον αφορά τη χρηστικότητα, διάδραση, κλπ, ή σε μελέτες  σχετικές με τα κοινωνικά δίκτυα που έχουν κάποια ιδιαίτερα χαρακτηριστικά (όπως  αυτά που υλοποιούν οι εφαρμογές του Ερωτ 1.1 );

Ευχαριστώ

**Comment by epidrome** [comment, marked as answer]
Αν το προηγούμενο υλικό μπορεί να επαναχρησιμοποιηθεί αυτούσιο τότε μάλλον δεν είναι μέρος της απάντησης αλλά θεωρητικό πλαίσιο. 

Οι γενικές αρχές σχεδίασης θα πρέπει να εστιάζονται ανάλογα με την εφαρμογή και τους χρήστες
